editors/vscode/package.json

```diff
@@ -66,6 +66,11 @@
                 "category": "Slint",
                 "icon": "$(preview)"
             },
+            {
+                "command": "slint.showPropertiesEditor",
+                "title": "Show Property Editor",
```

Comment:
Do we really need to add that in the command palette?  is there not other standard ways to show panes? Do other extension also have a command?
In fact, you can already do that by typing  "Explorer: Focus on Properties View"
(perhaps this tab should be renamed to "Slint Properties")